tests/utilities/helper.py

```diff
@@ -17,6 +17,20 @@ def get_random_name(length):
 
 def disable_telemetry(test_function):
     def wrapper(*args, **kwargs):
+        print("Disabling Telemetry.")
         os.environ["AZURE_CORE_COLLECT_TELEMETRY"] = "no"
         test_function(*args, **kwargs)
     return wrapper
+
+
+def set_authentication(test_function):
+    def wrapper(*args, **kwargs):
+        print("Setting auth for test run.")
+        pat_from_env = os.environ.get(PAT_ENV_VARIABLE_NAME)
+        if not pat_from_env:
+            print("PAT not found in environment variable. Falling back to hardcoded token.")
+            os.environ[PAT_ENV_VARIABLE_NAME] = DEVOPS_CLI_TEST_PAT_TOKEN
```

Comment:
do we need this?

Reply:
Actually No.. I was discussing this with Vladimir too regarding getting rid of the PAT in the file which can get accidently raised in the PR and be public. 

Should we set it to hardcoded PAT so that pipelines running recorded tests do not need to change else we will have to edit all pipelines to have a PAT env which can be any random value.